tests/regression/74-use_after_free/11-wrapper-funs-uaf.c

```diff
@@ -27,12 +27,14 @@ int main(int argc, char const *argv[]) {
     my_free2(p);
 
     *(p + 42) = 'c'; //WARN
-    printf("%s", p); //WARN
-
-    char *p2 = p; //WARN
+    // No dereferencing in the line below => no need to warn
+    printf("%s", p); //NOWARN
```

Comment:
Doesn't the %s implicitly dereference here?

Reply:
I believe it could be considered a dereference for reading the pointer's contents.

Reply:
I looked into a `printf` implementation and it looks like the pointer is indeed dereferenced.

Reply:
`printf` should now also be treated as an implicit dereference (cf. https://github.com/goblint/analyzer/pull/1123/commits/66f0c9da414be2dc0d989a9dd22db35978bf00ef)